### PR TITLE
Refs #13607 - Removed pulp.conf

### PR DESCRIPTION
**Pull request by parthaa**
The template is now removed and centralized in puppet-pulp

Check https://github.com/Katello/puppet-pulp/pull/124

### CODE REVIEW

**Comment by ekohl** [comment]
You can also remove the ```$apache_version``` from config.pp

**Comment by ekohl** [comment]
And while you're at it, I think you can also remove the include above it, but do double check that.

**Comment by parthaa** [comment]
> You can also remove the $apache_version from config.pp

good catch. Will do thanks!

File: manifests/init.pp [hunk 1]
```diff
@@ -140,6 +140,7 @@
     enable_puppet          => true,
     enable_docker          => true,
     enable_parent_node     => true,
+    ca_cert                => $::certs::ca_cert,
```
Comment (ehelms) [inline manifests/init.pp]: This doesn't need tobe set -- see https://github.com/Katello/puppet-katello/pull/102

File: manifests/config.pp [hunk 2]
```diff
@@ -1,10 +1,5 @@
 # Configuration for Katello development
 class katello_devel::config {
-
-  # Required until the katello.yml.erb changes to '@variable' syntax
-  include ::katello::params
```
Comment (ehelms) [inline manifests/config.pp]: I think we still need this as indicated by the comment

Comment (ehelms) [inline manifests/config.pp]: I take that back, @ekohl is right we updated katello.yaml properly so this is good to go

**Comment by ehelms** [comment]
ACK - thanks @parthaa !

**Pull request merged**